scss/_patterns_code-copyable.scss

```diff
@@ -20,13 +20,13 @@ $cc-button-size: 36px;
 
     &::before {
       color: $color-mid;
-      content: '$';
+      content: url('#{$assets-path}27e7d804-code-snippet.svg');
```

Comment:
We shouldn't make Vanilla patterns depend on external resources. If we need this icon it should be added to Vanilla icon mixins as data URL.

Reply:
Ah ok - updated!